### PR TITLE
Py3 refactore laposte

### PR DESCRIPTION
**Pull request by bealdav**
Remplace https://github.com/akretion/roulier/pull/118

### CODE REVIEW

File: tests/test-roulier-02.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+if carrier_obj.manage_multi_label or len(parcels) == 1:
+    payload = encoder.encode(data)
+    response = transport.send(payload)
+    decoder.decode(response, payload)
+else:
+    for parcel in parcels:
+        data["parcels"] = [parcel]
+        payload = encoder.encode(data)
+        response = transport.send(payload)
+        decoder.decode(response, payload)
```
Comment (florian-dacosta) [inline tests/test-roulier-02.py:60]: I don't understand why all the `get_label` logic is reimplented in tests?

The test should be as simple as possible and be something like this : 
`roulier.get("laposte_fr", "get_label")`
Then, check all the stuff you need to check...As this is the call that external application should do.
Or I missed something?

Same for other test cases

Comment (bealdav) [reply tests/test-roulier-02.py:60]: I'll remove this file

Comment (florian-dacosta) [reply tests/test-roulier-02.py:60]: It is the same for test 03 and 04

Comment (bealdav) [reply tests/test-roulier-02.py:60]: Yes the only valuable tests are in roulier/carriers/laposte_fr/tests/test_laposte.py
Sorry for the noise

**Comment by bealdav** [comment]
Replaced by 

https://github.com/akretion/roulier/pull/123 and
https://github.com/akretion/roulier/pull/124

cc @tonygalmiche

**Pull request closed without merging**